app/controllers/requests_controller.rb

```diff
@@ -9,8 +9,8 @@ class RequestsController < ApplicationController
 
   def index
     @filter = filter_param
-    @sent_requests_count = Request.include_associations.broadcasted.count
-    @planned_requests_count = Request.include_associations.planned.count
```

Comment:
I don't know why I added this scope here, but it doesn't make any sense now. We simply want the count so we don't need to include, preload, or eager load anything.